.github/workflows/todo_app_ci_pipeline.yml

```diff
@@ -11,11 +11,35 @@ on:
 
 jobs:
   build:
-    name: Build and test
+    name: Build & Test
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - run: docker build --target test --tag my-test-image .
     - run: docker run --env-file .env.test my-test-image todo_app/tests
     - run: docker run -e trello_board_id=${{secrets.TRELLO_BOARD_ID}} -e trello_key=${{secrets.TRELLO_KEY}} -e trello_token=${{secrets.TRELLO_TOKEN}}  my-test-image todo_app/e2e_tests
 
+  publish:
+    name: Push Image to DockerHub
+    needs: build
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/exercise-8' && github.event_name == 'push'
```

Comment:
Don't forget to update this when you merge 😄

Reply:
will merge and then update on master before I branch again